platforms/documentation/docs/src/docs/userguide/reference/best-practices/best_practices_performance.adoc

```diff
@@ -15,6 +15,62 @@
 [[best_practices_for_performance]]
 = Best Practices for Performance
 
+[[prefer_bin_distribution]]
+== Prefer the `-bin` Gradle Distribution
+
+Gradle publishes two distribution variants for each release: `-bin` (binaries only) and `-all` (binaries, sources, and documentation).
+For most builds, you should prefer the smaller `-bin` distribution.
+
+Using `-bin` reduces download size and verification effort, speeds up CI and developer builds, and limits the number of artifacts you need to trust.
+
+=== Explanation
+
+Each Gradle release provides:
+
+- `gradle-<version>-bin.zip` – binaries only
+- `gradle-<version>-all.zip` – binaries plus sources and offline documentation
+
+In modern setups:
+
+- IDEs and build tools can download sources and documentation directly from repositories or online docs.
+- The `-all` distribution is rarely required outside of specific offline or air-gapped environments.
+
+Preferring `-bin` helps because:
+
+- It **reduces download and cache size** for CI and local builds.
+- There is **less to verify and fewer artifacts to trust** (one smaller archive instead of a larger “everything included” zip).
+- It **shortens the feedback loop** when upgrading Gradle or rotating caches.
```

Comment:
What "rotating caches" do you have in mind here? I don't fully understand that part.